### PR TITLE
Fix server config scheme missing

### PR DESCRIPTION
**Pull request by yap12138**
Fix: missing udpPort when send beat info & lose server config scheme when called refreshServerIfNeed

### CODE REVIEW

**Comment by sanxun0325** [comment]
@yap12138 You can describe what the PR is trying to solve ，How does the problem reproduce and how does it verify.

**Comment by yap12138** [comment]
> @yap12138 You can describe what the PR is trying to solve ，How does the problem reproduce and how does it verify.

When i register instance using sdk, it will not report udpPort to server.  This caused i can't receive services change event by udp pushing.

**Comment by sanxun0325** [comment]
> When i register instance using sdk, it will not report udpPort to server. This caused i can't receive services change event by udp pushing.

This issue has been resolved in version v1.0.1.
Which version of SDK are you using。
pr: https://github.com/nacos-group/nacos-sdk-go/pull/142/files

**Comment by yap12138** [comment]
> > When i register instance using sdk, it will not report udpPort to server. This caused i can't receive services change event by udp pushing.
> 
> This issue has been resolved in version v1.0.1.
> Which version of SDK are you using。
> pr: https://github.com/nacos-group/nacos-sdk-go/pull/142/files

well...i'm using v1.0.0, This was already fixed.
But Scheme field in server config will be set empty while calling refreshServerIfNeed

File: common/nacos_server/nacos_server.go [hunk 1]
```diff
@@ -276,7 +276,7 @@ func (server *NacosServer) refreshServerSrvIfNeed() {
 					continue
 				}
 			}
-			servers = append(servers, constant.ServerConfig{IpAddr: splitLine[0], Port: uint64(port), ContextPath: constant.WEB_CONTEXT})
+			servers = append(servers, constant.ServerConfig{Scheme: constant.DEFAULT_SERVER_SCHEME, IpAddr: splitLine[0], Port: uint64(port), ContextPath: constant.WEB_CONTEXT})
```
Comment (sanxun0325) [inline common/nacos_server/nacos_server.go:279]: refreshServerIfNeed 中没有设置scheme，可以单独提一个pr修复。

Comment (yap12138) [reply common/nacos_server/nacos_server.go:279]: 可以的😀

Comment (sanxun0325) [reply common/nacos_server/nacos_server.go:279]: 我看了一下这个Scheme不需要，你这个endpoint也是通过nginx暴露出来的嘛。配置的ip列表前面可以加scheme.
如果配置的ip列表前面没有加scheme,那么需要在getAddress func()中做默认处理。

**Review by sanxun0325 — APPROVED** [review]
LGTM

**Comment by sanxun0325** [comment]
Thank you for your contribution and look forward to more

**Pull request merged**